ntttcp/ntttcp.TESTGEN.ps1

```diff
@@ -109,12 +111,13 @@ function banner {
 function test_ntttcp {
     [CmdletBinding()]
     Param(
-        [parameter(Mandatory=$true)] [String] $OutDir
+        [parameter(Mandatory=$true)] [String] $OutDir,
+        [parameter(Mandatory=$true)] [ValidateScript({Test-Path $_})] [String] $ConfigFile
     )
 
-    # execution time in seconds
-    [int] $g_runtime = 10
-    [int] $g_ptime   = 2
+    #Load the variables needed to generate the commands
+    . .\$ConfigFile
```

Comment:
What is the config file is not available/missing? 
Should there be safe defaults?  
Should we generate an error?  
Does the documentation detail that a config file and associated format is required?